iosxr_setup.py

```diff
@@ -36,6 +36,24 @@ def open_sessions(self, username, password):
         self.xr_telnet_sessions.append(self.xr1)
 
     #
+    # Send a global-vrf XR Linux command - wait for prompt
+    # Could do bash -c or run ip netns exec global-vrf bash
+    #
+    def send_operns(self, command):
+        xr1 = self.xr1
+
+        xr1.send("bash -c %s" % command)
+        time.sleep(1)
+        return xr1.wait("[$#]")
+
+    #
+    # Send a xrnns XR Linux command
+    #
+    def send_xrnns(self, command):
+        xr1 = self.xr1
+        xr1.send("run %s" % command)
+
```

Comment:
Seemed to mess with the resolve.conf.

I'm going to rework this code using sudovi.